package.json

```diff
@@ -39,7 +39,6 @@
     "concat-stream": "^1.5.1",
     "coveralls-send": "0.0.2",
     "cross-spawn-async": "^2.1.9",
-    "documentjs": "^0.4.1",
```

Comment:
i'm assuming this is going to be done with a plugin, is there an existing issue for that?